### PR TITLE
Homework 2

### PR DESCRIPTION
**Pull request by tribaeka**
Use `npm run test` to execute the usage.ts file.

### CODE REVIEW

File: homework-2/src/game-components/Inventory.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+export class Inventory {
+    private items: Item[] = [];
+
+    public addItem(item: Item): void {
+        this.items.push(item);
+    }
+
+    public sort(comparator?: IItemComparator): void {
+        if (comparator) {
```
Comment (AnastasiaGritsuk) [inline homework-2/src/game-components/Inventory.ts]: you can use ternary operator here:
this.items = comparator ? this.items.sort(comparator.compare) : sortBy(this.items, 'value');

File: homework-2/src/interfaces/IItemComparator.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { Item } from '../Item';
```
Comment (AnastasiaGritsuk) [inline homework-2/src/interfaces/IItemComparator.ts]: seems import is not correct
for me works this: import { Item } from '../game-components/Item';

File: homework-2/src/game-components/Inventory.ts (hunk 1 above)
Comment (tribaeka) [reply homework-2/src/game-components/Inventory.ts]: Fixed, thanks

File: homework-2/src/interfaces/IItemComparator.ts (hunk 2 above)
Comment (tribaeka) [reply homework-2/src/interfaces/IItemComparator.ts]: Fixed

**Pull request merged**